intranethome/views.py

```diff
@@ -0,0 +1,254 @@
+from django.shortcuts import render
+import json
+import re
+from django.http import Http404
+from wagtail.models import Site
+from base.wagtail_hooks import (get_required_groups, has_permission, redirect_users_without_permissions
+)
+from library_website.settings import MAIL_ALIASES_PATH
+
+
+# purpose of function: reading json text and converting to python
+# input: name of file
+# output: python text from the read in file
```

Comment:
All of these should be docstrings in the same style as in other places in the code. There's more info about that here: https://docs.google.com/document/d/1Myp9KYrNzZzhQvAF3tv9di4zgVL-T2jTwQ3gpfwKXnc/edit?usp=sharing.